### ISSUE TITLE
SEGUID Calculator: Move source code to a separate repo

### ISSUE THREAD

**Issue by HenrikBengtsson** [seguid/seguid.org#5]
<https://github.com/seguid/seguid.org/blob/main/web.qmd> is a nice example of how to use the JavaScript implementation of SEGUID.

I think its source code should live elsewhere; in either (a) stand-alone repo (e.g. seguid-calculator), or (b) be an "example app" under the <https://github.com/seguid/seguid-javascript> repo (e.g. subfolder apps/).  There's also enough additional JavaScript `<script>...</script>` code to make that code available via `<script src=.../>`, so that the webpage (web.qmd) source code becomes even smaller.  Then the SEGUID Calculator page could also add a link to the source code for anyone who's interested. This way, it would be as simple as possible for someone to re-use it elsewhere, tweak it, and contribute back to it.

Not sure how to provide access to the SEGUID Calculator on the website; maybe a new "Apps" menu with a 'SEGUID Calculator".  One could imagine other implementations or alternative in this 'Apps' menu, e.g. a Pyodide-based ShinyLive (https://shinylive.io/py/) app, or a webR-based ShinyLive (https://shinylive.io/r/examples/) app.  We might merge 'Live Demo' and 'Apps' into one menu.

**Comment by HenrikBengtsson** [comment]
Yes, it should be available there for end-users, but I'm saying the source code should live elsewhere.

**Comment by louisabraham** [comment]
Agreed!

How can we then include an external html in the Quatro build process?

**Comment by louisabraham** [comment]
awesome thanks
